sonar-db/src/main/java/org/sonar/db/permission/PermissionTemplateDao.java

```diff
@@ -218,34 +218,51 @@ public void updatePermissionTemplate(Long templateId, String templateName, @Null
     }
   }
 
+  /**
+   * @deprecated since 5.2 use the method with dbSession instead
+   */
+  @Deprecated
   public void insertUserPermission(Long templateId, Long userId, String permission) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)